public/app/app.config.js

```diff
@@ -0,0 +1,28 @@
+/*!
```

Comment:
@nicolaasmatthijs interesting, thanks for the reply! Does this mean that the outputted, minified js won't have any license header at all?

Perhaps we should have 1 file that has the `@preserve` or `@license` flag? https://github.com/mishoo/UglifyJS2#keeping-copyright-notices-or-other-comments